client/src/i18n/fr/report.json

```diff
@@ -136,7 +136,17 @@
       "TITLE" : "Rapport de changement d'inventaire",
       "DESCRIPTION": "Ce rapport montre les changements survenus sur les articles en stock au cours d'une période"
     },
-
+    "LOST_STOCK_REPORT": {
+      "TITLE": "Rapport sur les stocks perdus",
+      "DEPOT_ROLE": "List stock transfer losses for this depot as",
```

Comment:
In French you can say: Lister les pertes de transfert de stock pour ce dépôt comme

Reply:
Fixed.  Thanks!